spec/shared_print/finder_spec.rb

```diff
@@ -20,6 +20,46 @@
     Cluster.collection.find.delete_many
   end
 
+  describe "find by phase(s)" do
+    let(:p1) { "2017-09-30" }
```

Comment:
I'm on the fence about duplicating the dates between `SharedPrint::Finder` and here, and wondering if we should refer to `SharedPrint::Finder.phases`. On the one hand, I like avoiding too much logic in tests; on the other hand, these are essentially magic values that could be constantized. I wonder if we could split the difference by having e.g. `SharedPrint::Finder.PHASE_ONE_DATE`, etc?

Reply:
Yeah, I was a bit on the fence here, and ended up doing two half solutions.
I agree that this could and should be neater. 
It could almost be its own class (e.g. `SharedPrint::Phases`), but anything that cares about phases is going to want a finder anyways...

Reply:
I think it's probably fine as-is for now.